hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiPartitionCreate.java

```diff
@@ -0,0 +1,138 @@
+import com.hedera.services.bdd.spec.HapiSpec;
+import com.hedera.services.bdd.spec.transactions.HapiTxnOp;
+import com.hedera.services.bdd.spec.transactions.TxnUtils;
+import com.hedera.services.bdd.suites.HapiSuite;
+import com.hederahashgraph.api.proto.java.HederaFunctionality;
+import com.hederahashgraph.api.proto.java.Key;
+import com.hederahashgraph.api.proto.java.TokenCreatePartitionDefinitionTransactionBody;
+import com.hederahashgraph.api.proto.java.TokenID;
+import com.hederahashgraph.api.proto.java.Transaction;
+import com.hederahashgraph.api.proto.java.TransactionBody;
+import com.hederahashgraph.api.proto.java.TransactionResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class HapiPartitionCreate extends HapiTxnOp<HapiPartitionCreate> {
+
+    static final Logger log = LogManager.getLogger(HapiPartitionCreate.class);
+
+    private String token;
+
+    public HapiPartitionCreate(final String token) {
+        this.token = token;
+    }
+
+    protected HapiPartitionCreate self() {
+        return this;
+    }
+
+    @Override
+    public HederaFunctionality type() {
+        return HederaFunctionality.TokenCreatePartition;
+    }
+
+    private Optional<String> parent_token_id = Optional.empty();
+    private Optional<String> memo = Optional.empty();
+    private Optional<String> name = Optional.empty();
+
+    public HapiPartitionCreate parent_token_id(final String parent_token_id) {
+        this.parent_token_id = Optional.of(parent_token_id);
+        return this;
+    }
+
+    public HapiPartitionCreate name(final String name) {
+        this.name = Optional.of(name);
+        return this;
+    }
+
+    public HapiPartitionCreate memo(final String memo) {
+        this.memo = Optional.of(memo);
+        return this;
+    }
+
+    @Override
+    protected long feeFor(final HapiSpec spec, final Transaction txn, final int numPayerKeys) throws Throwable {
+        return HapiSuite.ONE_HUNDRED_HBARS;
+    }
+
+    @Override
+    protected Consumer<TransactionBody.Builder> opBodyDef(final HapiSpec spec) throws Throwable {
+        var id = TxnUtils.asTokenId(token, spec);
+        final TokenCreatePartitionDefinitionTransactionBody opBody = spec.txns()
+                .<TokenCreatePartitionDefinitionTransactionBody, TokenCreatePartitionDefinitionTransactionBody.Builder>
+                        body(TokenCreatePartitionDefinitionTransactionBody.class, b -> {
+                    b.setParentTokenId(id);
+                    name.ifPresent(b::setName);
+                    memo.ifPresent(s -> b.setMemo(s));
+                });
+        return b -> b.setTokenCreatePartition(opBody);
+    }
+
+    protected List<Function<HapiSpec, Key>> defaultSigners() {
+        List<Function<HapiSpec, Key>> signers = new ArrayList<>();
```

Comment:
This list should also contain `effectivePayer` like [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnUtils.java#L134), since the payer for the transaction also need to sign.